plenum/test/replica/test_replica_removing.py

```diff
@@ -0,0 +1,157 @@
+import pytest
+
+from plenum.test.delayers import cDelay
+from plenum.test.node_catchup.helper import waitNodeDataEquality
+from plenum.test.node_catchup.test_config_ledger import start_stopped_node
+from plenum.test.pool_transactions.helper import reconnect_node_and_ensure_connected, \
+    disconnect_node_and_ensure_disconnected
+from plenum.test.stasher import delay_rules
+from stp_core.loop.eventually import eventually
+from stp_core.common.log import getlogger
+from plenum.test.helper import sdk_send_random_requests, sdk_get_replies, sdk_send_random_and_check, waitForViewChange
+from plenum.test.test_node import ensureElectionsDone, checkNodesConnected
+
+from plenum.test.checkpoints.conftest import chkFreqPatched
+
+logger = getlogger()
+CHK_FREQ = 1
+
+
+def test_replica_removing(looper,
+                          txnPoolNodeSet,
+                          sdk_pool_handle,
+                          sdk_wallet_client):
+    """
+    1. Remove replica
+    2. Ordering
+    3. Check monitor and replicas count
+    """
+    node = txnPoolNodeSet[0]
+    start_replicas_count = node.replicas.num_replicas
+    index = start_replicas_count - 1
+    node.replicas.remove_replica(index)
+    _check_replica_removed(node, start_replicas_count)
+    # trigger view change on all nodes
```

Comment:
Such the teardown sections in tests make them dependent on each other. Instead of this introduce a fixture which ensures view change and use this fixture in every test.

Reply:
Done